diff --git a/apps/interpreters/micropython/Makefile b/apps/interpreters/micropython/Makefile
index e4b1b7f..fec76d9 100644
--- a/apps/interpreters/micropython/Makefile
+++ b/apps/interpreters/micropython/Makefile
@@ -69,13 +69,13 @@ CROSS_COMPILE ?= $(CROSSDEV)
 
 QSTR_DEFS = qstrdefsport.h
 
-# include py core make definitions
+# Include py core make definitions
 
 -include $(MICROPTHYON_SRCDIR)/py.mk
 
-# c99 is needed for micropython
+# c99 is needed for Micro Python
 
-CFLAGS+=-std=c99 -Wno-shadow -Wno-strict-prototypes
+CFLAGS += -std=c99 -Wno-shadow -Wno-strict-prototypes
 
 INC =  -I. -I$(WD) -I$(MICROPYTHON_UNPACKDIR) -I$(PY_SRC) -I$(BUILD)
 
diff --git a/nuttx/configs/Kconfig b/nuttx/configs/Kconfig
index 33ad22b..de74568 100644
--- a/nuttx/configs/Kconfig
+++ b/nuttx/configs/Kconfig
@@ -559,16 +559,6 @@ config ARCH_BOARD_PIRELLI_DPL10
 		This directory contains the board support for Pirelli dpl10 phones.  The
 		additions were made by Craig Comstock (with help form Alan Carvalho de Assis).
 
-config ARCH_BOARD_PX4FMU_V2
-	bool "PX4FMU v2"
-	depends on ARCH_CHIP_STM32F427V
-	---help---
-		This is a minimal configuration that supports low-level test of the
-		PX4FMU v2 in the NuttX source tree.  If you are using PX4, you
-		probably don't want this configuration.  You probably want the
-		latest supported configuration available from the PX4 GIT
-		repositories.
-
 config ARCH_BOARD_NUCLEO_F401RE
 	bool "STM32F401 Nucleo F401RE"
 	depends on ARCH_CHIP_STM32F401RE
@@ -1095,7 +1085,6 @@ config ARCH_BOARD
 	default "pic32-starterkit"         if ARCH_BOARD_PIC32_STARTERKIT
 	default "pic32mx7mmb"              if ARCH_BOARD_PIC32MX7MMB
 	default "pirelli_dpl10"            if ARCH_BOARD_PIRELLI_DPL10
-	default "px4fmu-v2_upstream"       if ARCH_BOARD_PX4FMU_V2
 	default "nucleo-f4x1re"            if ARCH_BOARD_NUCLEO_F401RE || ARCH_BOARD_NUCLEO_F411RE
 	default "qemu-i486"                if ARCH_BOARD_QEMU_I486
 	default "rgmp"                     if ARCH_BOARD_RGMP
@@ -1359,9 +1348,6 @@ endif
 if ARCH_BOARD_NUCLEO_F401RE || ARCH_BOARD_NUCLEO_F411RE
 source "configs/nucleo-f4x1re/Kconfig"
 endif
-if ARCH_BOARD_PX4FMU_V2
-source "configs/px4fmu-v2_upstream/Kconfig"
-endif
 if ARCH_BOARD_QEMU_I486
 source "configs/qemu-i486/Kconfig"
 endif
diff --git a/nuttx/configs/README.txt b/nuttx/configs/README.txt
index 9d09fe7..2092bab 100644
--- a/nuttx/configs/README.txt
+++ b/nuttx/configs/README.txt
@@ -454,12 +454,6 @@ configs/pirelli_dpl10
   It is a variant of the compal_e88 config with the small changes for the
   differences in the board.
 
-config/px4fmu-v2_upstream
-  This is a minimal configuration that supports low-level test of the
-  PX4FMU v2 in the NuttX source tree.  If you are using PX4, you probably
-  don't want this configuration.  You probably want the latest supported
-  configuration available from the PX4 GIT repositories.
-
 configs/qemu-i486
   Port of NuttX to QEMU in i486 mode.  This port will also run on real i486
   hardwared (Google the Bifferboard).
